examples/list_interfaces.rs

```diff
@@ -10,16 +10,11 @@
 
 extern crate pnet_datalink;
 
+
 fn main() {
     for interface in pnet_datalink::interfaces() {
-        let mac = interface.mac.map(|mac| mac.to_string()).unwrap_or("N/A".to_owned());
-        println!("{}:", interface.name);
-        println!("  index: {}", interface.index);
-        println!("  flags: {}", interface.flags);
-        println!("  MAC: {}", mac);
-        println!("  IPs:");
-        for ip in interface.ips {
-            println!("    {:?}", ip);
+        if interface.flags > 0 {
+            println!("{}", interface);
```

Comment:
Sorry for not finding this earlier. But why only print the interface if it has flags? Would it not make sense to list all interfaces, even those without flags? That is what we do before this PR.

Reply:
beacuse i think that mean nothing.

your are right, for a lib, list all interfaces is make sense.